### PR TITLE
bump version to 0.17.1

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ requirements:
     - bokeh
     - cloudpickle >=0.2.1
     - dask-core {{ version }}
-    - distributed >=1.21.0
+    - distributed >=1.21.1
```
Comment (jakirkham) [inline recipe/meta.yaml]: What causes this constraint? Also should it be in `setup.py`'s requirements for `dask`?

Comment (mrocklin) [reply recipe/meta.yaml]: It's not really a constraint, no more than the dask-core version number.  I figured this was typical for a metapackage to more strongly suggest version numbers

Comment (jakirkham) [reply recipe/meta.yaml]: If it's not a hard constraint, would it be ok for us to relax it?

Typically we try to mirror the requirements in `setup.py`. However things like extra requirements are added at people's discretion. The rules can be a bit looser with metapackages particularly if there is not a source of truth which it is tracking.

For context, was wanting to pin an older version of `distributed` so as to workaround issue ( https://github.com/dask/distributed/issues/1792 ) in the interim.

**Comment by mrocklin** [comment]
Sure.  Happy to relax as you see fit.  Your experience far outweighs my own
here :)

Short term you might also depend on dask-core rather than dask

**Comment by jakirkham** [comment]
Thanks. Addressing in PR ( https://github.com/conda-forge/dask-feedstock/pull/35 ).

Yeah, have been thinking about doing that as well.

**Pull request merged**